void/tests/test_math.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+
+from void import math_utils
+
+
+class CalculatePolyTest(unittest.TestCase):
+    def test_calculate_poly(self):
+        values = math_utils.calculate_poly((0, 0), 3, 3, 315)
+        expected = [[-1.29893408e-16, -2.12132034e+00],
+                    [-2.12132034e+00, 0.00000000e+00],
+                    [1.29893408e-16, 2.12132034e+00],
+                    [2.12132034e+00, 0.00000000e+00]]
```

Comment:
We could try to round those numbers so that we are not dealing with `e-16` which is really a `0`.
Maybe  calling `round(x, 4)` on each of those values:
```python
values = math_utils.calculate_poly((0, 0), 3, 3, 315)
values = [list(map(lambda x: round(x, 4), x)) for x in values]  # just rounding to 3 decimals
expected = [[0, -2.1213],
            [-2.1213, 0],
            [0, 2.1213],
            [2.1213, 0]]
```